justfile

```diff
@@ -26,6 +26,15 @@ oak_restricted_kernel_bin:
 _wrap_kernel kernel_bin_prefix:
     env --chdir=oak_restricted_kernel_wrapper OAK_RESTRICTED_KERNEL_FILE_NAME={{kernel_bin_prefix}}_bin cargo build --release
     rust-objcopy --output-target=binary oak_restricted_kernel_wrapper/target/x86_64-unknown-none/release/oak_restricted_kernel_wrapper oak_restricted_kernel_wrapper/target/x86_64-unknown-none/release/{{kernel_bin_prefix}}_wrapper_bin
+    rm -rf oak_restricted_kernel_wrapper/target/released_bin_with_components_{{kernel_bin_prefix}}
```

Comment:
I am not sure whether we should be moving / changing stuff in the Rust target dir. Maybe we should just do it outside, in some `out` dir or similar?

Reply:
probably, yes